### PR TITLE
DDT-23071 Discharge Wizard add of new Airforce portal.

### PR DESCRIPTION
**Pull request by zacharymorel**
## Description
Adding new Air Force Portal for BCMR (Board for Correction of Military Records) reviews. 

## Original issue(s)
department-of-veterans-affairs/va.gov-team#23071

## Testing done
Added a unit tests and ran it locally.

## Screenshots
Screen Shot 2021-07-21 at 2 03 34 PM

## Acceptance criteria
- [x] Link shows for Air Force branch only and review to BCMR

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/discharge-wizard/components/AirForcePortalLink.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const AirForcePortalLink = () => (
+  <section className="vads-u-margin-bottom--9">
+    <a
+      className="airForce-portal-link vads-u-display--flex vads-u-align-items--center vads-u-text-decoration--none"
+      href="https://afrba-portal.cce.af.mil/#portal"
```
Comment (kelsonic) [inline src/applications/discharge-wizard/components/AirForcePortalLink.jsx:7]: I think we need rel="noreferrer noopener" here 🙂 also do you want this to be a target="_blank"?

File: src/applications/discharge-wizard/components/gpMinorComponents/ResultsSummary.jsx [hunk 2]
```diff
@@ -3,18 +3,31 @@ import React from 'react';
...
   )}${forReconsideration ? ' for reconsideration' : ''}`;
 
+  if (airForceAFRBAPortal) {
+    summary =
+      'Based on your answers, you need to complete an Application for Correction of Military Record (DD 149). You can download this form from the Air Force Review Agency Portal';
+  }
+
   return (
     <section className="va-introtext">
       <p>{summary}.</p>
```
Comment (kelsonic) [inline src/applications/discharge-wizard/components/gpMinorComponents/ResultsSummary.jsx:33]: Should we include the period when declaring `summary` above?

File: src/applications/discharge-wizard/components/AirForcePortalLink.jsx (hunk 1 above)
Comment (zacharymorel) [reply src/applications/discharge-wizard/components/AirForcePortalLink.jsx:7]: I'll add the `rel="noreferrer noopener"`. That was to tell the other website Analytics that it's direct traffic? My memory does not serve me well on why we use the attribute.

As far as opening a new tab, a11y team requested that it doesn't open a tab and we navigate to the new portal.

File: src/applications/discharge-wizard/components/gpMinorComponents/ResultsSummary.jsx (hunk 2 above)
Comment (zacharymorel) [reply src/applications/discharge-wizard/components/gpMinorComponents/ResultsSummary.jsx:33]: Nice catch, I can change that bit.

File: src/applications/discharge-wizard/components/AirForcePortalLink.jsx (hunk 1 above)
Comment (zacharymorel) [reply src/applications/discharge-wizard/components/AirForcePortalLink.jsx:7]: Oh this explanation helps a bit. 
https://stackoverflow.com/a/50709724

File: src/applications/discharge-wizard/sass/discharge-wizard.scss [hunk 3]
```diff
@@ -90,4 +91,8 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
       }
     }
   }
+
+  .airForce-portal-link {
+    width: max-content;
```
Comment (kelsonic) [inline src/applications/discharge-wizard/sass/discharge-wizard.scss:96]: Have you tried using `vads-u-width--full`?

**Review by kelsonic — APPROVED** [review]
lgtm 🎉

**Review by k80bowman — APPROVED** [review]

File: src/applications/discharge-wizard/components/AirForcePortalLink.jsx (hunk 1 above)
Comment (zacharymorel) [reply src/applications/discharge-wizard/components/AirForcePortalLink.jsx:7]: Looks like we are gtg for links not opening in new tabs!

**Review by k80bowman — APPROVED** [review]

**Pull request merged**